dex/src/main/scala/com/wavesplatform/dex/AddressStateDiff.scala

```diff
@@ -0,0 +1,15 @@
+package com.wavesplatform.dex
+
+import com.wavesplatform.dex.domain.asset.Asset
+import com.wavesplatform.dex.domain.utils.ScorexLogging
+
+case class AddressStateDiff(changedSpendableAssets: Set[Asset], changedReservableAssets: Set[Asset]) extends ScorexLogging {
+
+  def getAllAssets: Set[Asset]                                  = changedSpendableAssets ++ changedReservableAssets
+  def updateReservedAssets(diff: Set[Asset]): AddressStateDiff  = copy(changedReservableAssets = changedReservableAssets ++ diff)
+  def updateSpendableAssets(diff: Set[Asset]): AddressStateDiff = copy(changedSpendableAssets = changedSpendableAssets ++ diff)
```

Comment:
Probably the word `append` would be better here?

Reply:
Done (_put_)